package.json

```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=6.9.5"
   },
-  "version": "8.0.0",
+  "version": "8.0.1",
```

Comment:
did you mean to version bump?

Reply:
Yes, since I am adding a new module and package lock. I want both yarn and npm 6 to be able to pick it up.